newIDE/app/src/ResourcesList/index.js

```diff
@@ -155,6 +155,25 @@ export default class ResourcesList extends React.Component<Props, State> {
     );
   };
 
+  _getResourceThumbnail = (resource: gdResource) => {
+    switch (resource.getKind()) {
+      case 'image':
+        return getLocalResourceFullPath(this.props.project, resource.getName());
+      case 'audio':
+        return 'res/actions/music24.png';
+      case 'json':
+        return 'res/actions/fichier24.png';
+      case 'video':
+        return 'JsPlatform/Extensions/videoicon24.png';
+      case 'font':
+        return 'res/actions/font24.png';
+      case 'bitmapFont':
+        return 'JsPlatform/Extensions/bitmapfont32.png';
+      default:
+        return 'res/unknown32.png'; //Icon type inconnu
```

Comment:
Comment in French. Remove it entirely, the "default" is already telling what's going on.